content/learn/book/getting-started/_index.md

```diff
@@ -2,6 +2,8 @@
 title = "Getting Started"
 template = "docs-section.html"
 insert_anchor_links = "right"
+# Redirect because page already shows in search results. Remove when developing.
```

Comment:
```suggestion
# Redirect since old links, and for a time search results,
# use this link for what is now the quick start guide.
# Remove the line below when developing to see the page render
# on Zola rather than redirecting to quick start guide.
# Please add it back before committing.
```
This should be a bit more descriptive.